sopel/plugin.py

```diff
@@ -896,6 +897,44 @@ def guarded(bot, trigger, *args, **kwargs):
     return actual_decorator
 
 
+def require_bot_privilege(level, message=None, reply=False):
+    """Decorate a function to require the bot a privilege level or above.
```

Comment:
```suggestion
    """Decorate a function to require a minimum channel privilege for the bot.
```

I even wrote this in my editor to make sure it would fit in 80 cols. 😛